src/backend/utils/vkbasalt.py

```diff
@@ -0,0 +1,257 @@
+# vkbasalt.py: library supplying the logics and functions to generate configs
+#
+# Copyright 2022 vkbasalt-cli Contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from os import path, environ, system, remove
+from sys import exit
+from shutil import copyfile
+import configparser
+import logging
+
+
+def parse(args, *arguments):
+    # Apply default settings if possible
+    if args.default:
+        install_paths = [
+            "/usr/lib/extensions/vulkan/vkBasalt/etc/vkBasalt",
+            "/usr/local",
+            "/usr/share/vkBasalt",
+        ]
+        for index, item in enumerate(install_paths):
+            file_path = path.join(item, "vkBasalt.conf")
+            if path.isfile(file_path):
+                if args.output:
+                    logging.info(f"Outputting file to {file_path}")
+                    copyfile(file_path, path.join(args.output, "vkBasalt.conf"))
+                if args.exec:
+                    environ["ENABLE_VKBASALT"] = "1"
+                    environ["VKBASALT_CONFIG_FILE"] = file_path
+                    system(f"{args.exec}")
+                return
+        logging.error(f"No such path for vkBasalt exists")
+        exit(1)
```

Comment:
In what context this runs? You shouldn't exit a GTK app this way.

Reply:
It's not supposed to run.

Reply:
but then why it is there?

Reply:
It's a library I wrote to make it universal in areas like the CLI. https://gitlab.com/TheEvilSkeleton/vkbasalt-cli